pkg/crc/ssh/client.go

```diff
@@ -68,11 +71,11 @@ func clientConfig(user string, keys []string) (*ssh.ClientConfig, error) {
 		Auth: []ssh.AuthMethod{ssh.PublicKeys(privateKeys...)},
 		// #nosec G106
 		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
-		Timeout:         10 * time.Second,
+		Timeout:         dialTimeout,
```

Comment:
this `dialTimeout` is not enough to avoid the issue, the connection is established and then it hangs?

Reply:
Still wondering about this :)

Reply:
> does not seem to have any impact on the hang which this PR is trying to fix

what is it trying to fix? can you explain what steps you did you play this scenario?